### DISCUSSION TITLE
How to stop CSS styling of a header applying to entire section?

### DISCUSSION THREAD

**Discussion by Hedonical** [quarto-dev/quarto-cli#2056] [Q&A]
Hi everyone,

I am trying to make it so only the header of a section is centered and center text as seen in the live editor view:
![Screenshot from 2022-08-22 11-11-08](https://user-images.githubusercontent.com/91704211/185992668-ea435ec6-3cce-40d4-b481-c867d3f87091.png)

But when I render the HTML document, it applys the centering and color to the entire section:
![Screenshot from 2022-08-22 14-24-18](https://user-images.githubusercontent.com/91704211/185992753-fd422d6e-5a8d-4162-947e-9a3ce3b2e9e4.png)

Is there an easy way to make it so the css elements only apply to the header and not all of the text under?

**Comment by cscheid** [comment]
@Hedonical Thanks. I've created an issue to track: #2057. Can you follow our github issue template and provide a reproducible example?
